<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IssueProperty</class>
 <widget class="QWidget" name="IssueProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>60</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="m_icon_label">
       <property name="text">
        <string>ICON</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="m_warning_label">
       <property name="text">
        <string>Warning</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="m_suggestion_label">
     <property name="text">
      <string>Suggestion</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
